mutiny-core/src/node.rs

```diff
@@ -242,6 +245,36 @@ impl Node {
         let channel_manager: Arc<PhantomChannelManager> =
             Arc::new(read_channel_manager.channel_manager);
 
+        // Check all existing channels against default configs.
+        // If we have default config changes, those should apply
+        // to all existing and new channels.
+        let default_config = default_user_config().channel_config;
+        for channel in channel_manager.list_channels() {
+            // unwrap is safe after LDK.0.0.109
+            if channel.config.unwrap() != default_config {
```

Comment:
nit: Could do `if channel.config != Some(default_config)` just to be safe but not a big deal